src/main/resources/sql/db.changelog-1.0.sql

```diff
@@ -0,0 +1,15 @@
+--liquibase formatted sql
+
+--changeset aziz:1
+CREATE TABLE IF NOT EXISTS movie (
+    id          BIGSERIAL PRIMARY KEY,
+    title       TEXT    NOT NULL CHECK (title <> ''),
+    description TEXT    NOT NULL CHECK (description <> ''),
+    produced_year INTEGER NOT NULL CHECK (year BETWEEN 1850 AND 2050),
```

Comment:
The check is done with a column named `year`, but the column is created as `produced_year`.